src/esw/brushed_motors/ROSHandler.cpp

```diff
@@ -43,95 +43,69 @@ void ROSHandler::init(ros::NodeHandle* rosNode) {
 
 // REQUIRES: nothing
 // MODIFIES: nothing
-// EFFECTS: Moves the RA joints in open loop
-// and publishes angle data right after.
-void ROSHandler::moveRA(const sensor_msgs::JointState::ConstPtr& msg) {
+// EFFECTS: Moves a controller in open loop.
+std::optional<float> ROSHandler::moveControllerOpenLoop(const std::string& name, float vel) {
```

Comment:
I understand what that vel is velocity after reading through the code but it took me a second. Maybe just make it velocity so it's super clear?

Reply:
yeah that makes sense

Reply:
fixed